mkosi/__init__.py

```diff
@@ -4504,7 +4504,9 @@ def generate_key_cert_pair(args: Args) -> None:
     keylength = 2048
     expiration_date = datetime.date.today() + datetime.timedelta(int(args.genkey_valid_days))
 
-    configdir = finalize_configdir(args)
+    configdir = finalize_configdir(args.directory)
+    if not configdir:
+        die("genkey cannot be used with --directory \"\"")
```

Comment:
Ruff complains about this and suggest
```suggestion
        die('genkey cannot be used with --directory ""')
```
I'd suggest
```suggestion
        die("genkey cannot be used with --directory ''")
```
since I'd type that (no need to press shift and for literals I don't want interpolation anyway) or
```suggestion
        die("genkey cannot be used with empty --directory)
```